src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -8059,7 +8059,7 @@ internal ParseError(IScriptExtent extent, string errorId, string message, bool i
         /// <returns></returns>
```

Comment:
@TylerLeonhardt, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 255
255:             $result[0] | Should -BeOfType Microsoft.PowerShell.Commands.TestConnectionCommand+TraceStatus

```